server.js

```diff
@@ -53,6 +59,36 @@ export default function (opt) {
   app.use(router.routes());
   app.use(router.allowedMethods());
 
+  router.del('/api/tunnels/:id', async (ctx, next) => {
```

Comment:
Will we keep this just in case or good to drop?

Reply:
It's nice to have.